docs/source/overview/core-concepts/sensors/visuo_tactile_sensor.rst

```diff
@@ -0,0 +1,188 @@
+.. _overview_sensors_tactile:
+
+.. currentmodule:: isaaclab
+
+Visuo-Tactile Sensor
+====================
+
+
+The visuo-tactile sensor in Isaac Lab provides realistic tactile feedback through integration with TacSL (Tactile Sensor Learning) [Akinola2025]_. It is designed to simulate high-fidelity tactile interactions, generating both visual and force-based data that mirror real-world tactile sensors like GelSight devices. The sensor can provide tactile RGB images, force field distributions, and other intermediate tactile measurements essential for robotic manipulation tasks requiring fine tactile feedback.
+
+
+.. figure:: ../../../_static/overview/sensors/tacsl_diagram.jpg
+    :align: center
+    :figwidth: 100%
+    :alt: Tactile sensor with RGB visualization and force fields
+
+
+Configuration
+~~~~~~~~~~~~~
+
+Tactile sensors require specific configuration parameters to define their behavior and data collection properties. The sensor can be configured with various parameters including sensor resolution, force sensitivity, and output data types.
+
+.. code-block:: python
+
+    from isaaclab.sensors.tacsl_sensor import VisuoTactileSensorCfg
+
+    tactile_sensor = VisuoTactileSensorCfg(
+        prim_path="{ENV_REGEX_NS}/Robot/tactile_sensor",
+        ## Sensor configuration
+        sensor_type="gelsight_r15",
+        enable_camera_tactile=True,
+        enable_force_field=True,
+        ## Elastomer configuration
+        elastomer_rigid_body="elastomer",
+        elastomer_tactile_mesh="elastomer/visuals",
+        elastomer_tip_link_name="elastomer_tip",
+        # Force field configuration
+        num_tactile_rows=20,
+        num_tactile_cols=25,
+        tactile_margin=0.003,
+        ## Indenter configuration (will be set based on indenter type)
+        indenter_rigid_body="indenter",
+        indenter_sdf_mesh="factory_nut_loose/collisions",
```

Comment:
What does the indenter configuration mean? Is it fixed to particular object you want the data for?